modules/core/src/main/java/org/apache/synapse/config/xml/InvokeMediatorFactory.java

```diff
@@ -74,6 +74,13 @@ protected Mediator createSpecificMediator(OMElement elem, Properties properties)
             throw new SynapseException(msg);
         }
 
+        OMAttribute errorHandlerAttr = elem.getAttribute(ATT_ONERROR);
+        if (errorHandlerAttr != null) {
+            if (StringUtils.isNotEmpty(errorHandlerAttr.getAttributeValue())) {
```

Comment:
Shall we combine two if conditions and also save errorHandlerAttr.getAttributeValue() to a value and re use